### ISSUE TITLE
Trouble with connexion to ARS

### ISSUE THREAD

**Issue by rutonyru** [johanwasserman/tRemedy#1]
Hi,

I'm trying to use tRemedy components to upload data in ars forms (with a 7.6.04 server). 
Everything seems to be Ok till I do it in two consecutive jobs. It's like the first job disabled the common user used on the ars server. I have to stop/start the ars server to make it 
Did you notice something special about this ?
Can you help me ?
Thanks,
Rutonyru.

**Comment by johanwasserman** [comment]
Can you send me your job (export to zip archive) so I can test it?  Remove
your Remedy config info if you are concerned about security.  I will gladly
make fixes or suggest a solution.

Kind regards,
Johan.

**Comment by rutonyru** [comment]
Well, I spent few hours to understand what could happens. I have 2 ARS environments and the "trouble" appears only in one of those. I checked a lot of things without really "relevant results". I decided to change the user used in my job (instead of Demo) and it works !
On monday, my ARS expert will come back...and will help me to explain the situation.
Till monday, I have a solution ! 
Thank you for your kind attention :)
Rutonyru
PS : Sorry for my pour english ;)
